modyn/config/schema/pipeline/sampling/downsampling_config.py

```diff
@@ -74,6 +77,7 @@ class GradMatchDownsamplingConfig(BaseDownsamplingConfig):
 
     strategy: Literal["GradMatch"] = "GradMatch"
     balance: bool = Field(False, description="If True, the samples are balanced.")
+    full_grad_approximation: FullGradApproximation = Field(default="LastLayer")
```

Comment:
Could you either here or at the definition on `FullGradApproximation` add a description/comment on the meaning of this such that future users understand what this is doing?

Reply:
Make sense. Done